examples/flax/language-modeling/run_t5_mlm_flax.py

```diff
@@ -476,12 +477,16 @@ def write_eval_metric(summary_writer, eval_metrics, step):
 
     # Load pretrained model and tokenizer
 
+    if model_args.model_type == "byt5":
+        tokenizer_type = ByT5Tokenizer
+    else:
+        tokenizer_type = T5TokenizerFast
     if model_args.tokenizer_name:
-        tokenizer = T5TokenizerFast.from_pretrained(
+        tokenizer = tokenizer_type.from_pretrained(
```

Comment:
Could we try to directly use the `AutoTokenizer` here instead?

Reply:
That makes sense, I'll make a new commit